x-pack/legacy/plugins/reporting/log_configuration.ts

```diff
@@ -6,21 +6,19 @@
 
 import getosSync, { LinuxOs } from 'getos';
 import { promisify } from 'util';
-import { ServerFacade, Logger } from './types';
+import { BROWSER_TYPE } from './common/constants';
+import { ReportingConfig } from './server';
+import { Logger } from './types';
 
 const getos = promisify(getosSync);
 
-export async function logConfiguration(server: ServerFacade, logger: Logger) {
```

Comment:
I'm not really sure why we have this file. It appears that we're logging the `disableSandbox` setting, in case we have made an auto-determination for that setting. But maybe we shouldn't be auto-determining it in the first place?

Reply:
I think we should probably still turn this on as much as we possibly can. It's generally recommended everywhere I've read about it, and if that means still dynamically setting it then it's probably for the better.

I do question the log of logging it if it cannot be set (IE their distro doesn't support it) since there's nothing actionable they can do.

Reply:
As far as I can tell, we do preach best practices in our documentation and tell people to leave the sandbox enabled.

But [the original schema had some code to do OS detection and automatically disable the sandbox if the OS is known to not be supported](https://github.com/elastic/kibana/pull/55882/files#diff-6f2b7382b8c186f44b8c74ab86ee4b85L104). That's the part I'm struggling with. The whole schema is re-written now, so if we continue the feature to automatically disable this, that part will have to go in `createConfig$`

Reply:
I just did a quick check on this PR branch and noticed that `default_chromium_sandbox_disabled.ts` is still in the repo as dead code